### ISSUE TITLE
Thumbnails for folders

### ISSUE THREAD

**Issue by melvin4u445** [ARK-Builders/ARK-Navigator#212]
We could show thumbnails for folders instead of a generic icon to make it easily identifiable for the user.
The thumbnail could be autogenerated based on the content inside the folder or could even be customizable so that the user can choose a thumbnail of their liking.

**Comment by kirillt** [comment]
Good idea. We also can implement small Gallery-like widgets on Folders screen displaying last or most scored resources.
